### DISCUSSION TITLE
Compatibility with Uno Platform..?

### DISCUSSION THREAD

**Discussion by IanMcCarthy** [shinyorg/shiny#764] [Q&A] [answered]
Hi,

I'm looking at several cross platform solutions for app development and am currently looking at Uno Platform.

Can you give any guidance as to the suitability of Shiny with Uno Platform..?

Thanks...

**Comment by aritchie** [comment, marked as answer]
Shiny will work on iOS, Android, & UWP on the .NET platform which means it will work on those same platforms on Uno
